Modules/_randommodule.c

```diff
@@ -526,12 +525,16 @@ random_new(PyTypeObject *type, PyObject *args, PyObject *kwds)
     self = (RandomObject *)PyType_GenericAlloc(type, 0);
     if (self == NULL)
         return NULL;
-    tmp = random_seed(self, args);
-    if (tmp == NULL) {
-        Py_DECREF(self);
-        return NULL;
+
+    if (PyTuple_GET_SIZE(args) == 1) {
```

Comment:
Correct me if I'm wrong, but afaik, random_new is called with args of _random.Random's init, which can contain more than 1 argument. So it might be more robust to check for len(args) > 0 instead.
```python
>>> import random
>>> random.Random(1,2,3)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: Random.__init__() takes from 1 to 2 positional arguments but 3 were given
>>>
>>> import _random
>>> _random.Random(1,2,3)
>>> # no error
```
```suggestion
    if (PyTuple_GET_SIZE(args) > 0) {
```

Reply:
@Fidget-Spinner thanks for the review! 
I think, in previous versions (e.g. 3.6) it just accepts one argument. So I'm not sure if it is a good behavior for `random_new` to accept more arguments or not! (I think, it should raise an exception in case of calling with 2 or more args)
also after [this commit](https://github.com/python/cpython/pull/13532/commits/ae62ca1bd50793c1a412a1f0197289d33f19de25) it is allowed to pass more arguments.
Dear @rhettinger, do you have any idea about it?

Reply:
Only accept one argument.  If there are more, raise a *TypeError*.